README.md

```diff
@@ -1,23 +1,27 @@
 # Calico Networking for rkt
```

Comment:
is it Rkt or rkt? Each are used in this doc, but would be nice to just use one of them.